packetbeat/protos/nfs/nfs4.go

```diff
@@ -167,18 +167,20 @@ func (nfs *Nfs) eatData(op int, xdr *Xdr) {
 
 func (nfs *Nfs) getV4Opcode(xdr *Xdr) string {
```

Comment:
Do we return the last seen name on purpose or do we want to return 'ILLEGAL' if no matching op has been found? Some test with 'ILLEGAL' opcode would be nice.

proposal for returning 'ILLEGAL' if no op has been found:

```
func (nfs *Nfs) getV4Opcode(cdr *Xdr) string {
    count := int(xdr.getUint())
    for i := 0; i < count; i++ {
        op := int(xdr.getUInt())
        name, ok := nfs_opnum4[op]
        if !ok {
            return fmt.Sprintf(...)
        }

        switch op {
          case ...:
            return name
          default:
            nfs.eatData(op, xdr)
        }
    }

  return "ILLEGAL"
}
```